_includes/sponsorship.html

```diff
@@ -122,6 +142,14 @@ <h3 class="center-text">Bronze Sponsors</h3>
 
 .logos-home > li {
   flex: 0 0 50%;
+  text-align: center;
+  font-size: var(--step-3);
+  font-weight: bold;
+}
+
+.logos-home > li > span {
+  display: inline-block;
+  margin: 1rem;
```

Comment:
Lines 150 - 152 should not be needed after removing span. The list items should all be treated the same because they are flex items.

Reply:
Removed.